components/Navbar.js

```diff
@@ -36,23 +36,24 @@ export default function Navbar() {
   const QUICKET_LINK = "https://qkt.io/pyconug";
   return (
     <nav className="sticky top-0 z-10 bg-white backdrop-filter backdrop-blur-lg bg-opacity-30 border-b border-gray-200">
-      <div className="max-w-5xl mx-auto px-4">
+      <div className="max-w-5xl mx-auto px-5">
         <div className="flex items-center justify-between h-16">
           <Link href={"/"}>
             <Image src={logo} alt="Pycon logo" width={120} height={70} />
           </Link>
-          <div className="md:flex space-x-4 text-gray-900 hidden">
-            {NavbarLinks.map(({ path, label }, index) => {
-              return <a key={index} href={path}>{label}</a>;
```

Comment:
for instance here, the `key` prop has been removed by the update

Reply:
Just noticed this! Thank you for resolving it! 🙏